db/schema.rb

```diff
@@ -16,8 +16,8 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "api_tokens", force: true do |t|
-    t.string   "access_token"
+  create_table "api_tokens", force: :cascade do |t|
+    t.string   "access_token", limit: 255
```

Comment:
Where are these limits coming from? Do we need a corresponding migration?

More info: https://github.com/rails/rails/pull/14579